### ISSUE TITLE
FastLane PDF Problem - High Importance

### ISSUE THREAD

**Issue by kmonterr** [18F/nsf-sbir#1331]
FYI - seems that the FastLane PDF won’t download or open? Please look into this issue ASAP, as the deadline is approaching and people need this.

Subject: PDF of Fastlane User Guide via Seed Fund does not appear to open
Good morning Amanda!
I was just going through some of the help info. provided for SBIR applicants in Research.gov and one of the Seed Fund URL's doesn't appear to be working (tried IE, Firefox, and Safari on my Mac and Jorge also tried from his end to no avail).
The URL the link below gets directed to is: https://seedfund.nsf.gov/assets/files/applicants/Phase_I_Proposal_Preparation_Booklet.pdf 
Can you please check this out from your end and let us know if you're having problems as well?

**Comment by seakhtar** [comment]
Hi @kmonterr ,

Is the above link not opening for you, or is there a different link you're referring to? When I click the above, the pdf does open for me. Do you get an error, or does it just appear to freeze? Since the file is quite big, it might be taking a while to download.

Let me know--

**Comment by deladie** [comment]
@kmonterr I just tested the link as well and it worked for me in Chrome and Firefox.  Also I visited https://seedfund.nsf.gov/apply/ and navigated down to 7 Start your application and selected the link to download the pdf there.  The link worked there as well.  We may need some more information to help replicate the problem.  If you share their contact info via email, we follow up to troubleshoot more.
